interpreter/value_storage_reference.go

```diff
@@ -467,3 +467,8 @@ func forEachReference(
 func (v *StorageReferenceValue) BorrowType() sema.Type {
 	return v.BorrowedType
 }
+
+func (v *StorageReferenceValue) Iterator(context ValueStaticTypeContext, locationRange LocationRange) ValueIterator {
+	//TODO implement me
```

Comment:
Same as for `StringValue.Iterator`

Reply:
https://github.com/onflow/cadence/pull/3825#discussion_r2024007432